connector/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM gradle:7.6.0-jdk17 AS build
 
+ARG EDC_LAST_COMMIT_INFO="The GitHub Pipeline will replace this string during build time with the commit information (SHA, author and commit message) of the built image."
```

Comment:
"The docker container was built outside of github actions and you didn't provide the build arg EDC_LAST_COMMIT_INFO_ARG, so there's no last commit info."

Also should contain a comment above, that we want to se the env var from the github action, but there's no way in the plugin to specify env vars, so we specify a build arg that we'll copy into the env var.

Reply:
done